### DISCUSSION TITLE
The Module is deprecated in version 2.0

### DISCUSSION THREAD

**Discussion by xxxpsyduck** [apache/incubator-mxnet#19216] [General]
Is there any documentation that can help me to migrate my current code using Module to something new in mxnet 2.0?

**Comment by szha** [comment]
The migration doc is planned for official release and AFAIK it hasn't been started yet.

There are several aspects to the migration from module:
- If you used Symbol for programming before, we recommend moving to Gluon for writing models instead.
- For executing a symbolic graph, gluon.SymbolBlock can be used for this purpose.
- For the Keras-like training pipeline management, we have a gluon.contrib.estimator that @roywei built. here's an example of the usage: https://github.com/apache/incubator-mxnet/blob/bd0846a69addf18529d5dcf856b28bed1a38cdbc/tests/nightly/estimator/test_estimator_cnn.py#L128-L156
